bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
-            <th>Type</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td><code>cov:methodName</code></td>
-            <td>CoAP method name (Literal).</td>
-            <td>optional</td>
-            <td>
-              <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a>
-              <p>(one of <code>"GET"</code> (1), <code>"POST"</code> (2), <code>"PUT"</code> (3), <code>"DELETE"</code> (4),
-                <code>"FETCH"</code> (5), <code>"PATCH"</code> (6), <code>"iPATCH"</code> (7))</p>
-            </td>
-          </tr>
-          <tr>
-            <td><code>cov:options</code></td>
-            <td>CoAP options sent with the message, e.g., <code>[ { "cov:optionName": "Accept", "cov:optionValue": 110 }]</code> to observe.</td>
-            <td>optional</td>
-            <td>
-              array of <code>cov:MessageOption</code>
-            </td>
-          </tr>
-          <tr>
-            <td><code>cov:optionName</code></td>
-            <td>Option name (Literal), see <a
-                href="https://www.iana.org/assignments/core-parameters/core-parameters.xhtml#option-names">CoRE
-                Parameters</a>.</td>
-            <td>mandatory within <code>cov:MessageOption</code></td>
-            <td>
-              <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#string"><code>string</code></a>
-            </td>
-          </tr>
-          <tr>
-            <td><code>cov:optionValue</code></td>
-            <td>Header value (Literal).</td>
-            <td>mandatory within <code>cov:MessageOption</code></td>
-            <td>
-              <a href="https://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyType"><code>anyType</code></a>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+    <section id="overview">
+        <h2>Protocol Overview</h2>
+        <p class="ednote">TODO: Provide an overview of the protocol and the protocol binding.</p>
+        <table class="def">
+            <thead>
+                <tr>
+                    <th>Feature</th>
+                    <th>Covered by</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Request to Server</td>
+                    <td><code>"href": ..., "coap:method": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Request to Forward Proxy</td>
+                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
+                </tr>
+                <tr>
+                    <td>Caching: Validation</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Conditional Requests</td>
+                    <td>n/a</td>
+                </tr>
+                <tr>
+                    <td>Content</td>
+                    <td><code>"coap:format": ...</code></td>
```

Comment:
Since it looks like we're going to change quite a bit, I'm wondering if we shouldn't rather intentionally change the prefix...

Reply:
Good point ;) It seems as if there was the time when this was already the case, though: https://github.com/w3c/wot-binding-templates/blob/e4fd7a4f925b7da063c6f99f9a1debb721a22d65/index.html I couldn't find the reason why the prefix was changed from `coap:` to `cov:` (probably because `http:` was changed to `htv:` in order to prevent name clashes...?) but I would also be in favor of using `coap:` to make it more clear which protocol is being referred to.

Reply:
this is a rather long discussion:
- https://github.com/w3c/wot-thing-description/issues/443
- In the modbus PR from @relu91 we had a discussion on this but I could not find it

Reply:
> What is format by the way? I find different meanings when looking at Wikipedia and rfc

A CoAP content format is the combination of an HTTP content type (such as `text/plain;charset=utf-8`) with an HTTP content coding (such as `deflate`). The CoAP Content-Format Option describes the content format of the payload in a request or response. The CoAP Accept Option in a request specifies the content format of the payload in a successful response.